### ISSUE TITLE
`pickerInput` ADA improvements

### ISSUE THREAD

**Issue by carlganz** [dreamRs/shinyWidgets#640]
Hello,

Upon ADA review, one of our applications was flagged because the `pickerInputs` do not have `aria` accessibility attributes setup for the `Select` and `Deselect` all buttons, so users are unable to access those buttons via the tab button. 

Ideally the aria posinset and set size attributes could be generated dynamically to include those buttons.

**Comment by carlganz** [comment]
Issue is actually upstream in `bootstrap-select` library [issue here](https://github.com/snapappointments/bootstrap-select/issues/2848)
